docs/configuring_keyword_alignment.rst

```diff
@@ -205,13 +205,14 @@ Rule Specific Keyword Alignment Configuration
           hold_transmission <= '1';
       end if;
 
-#. :code:`case_control_statements_end_group` - if set to :code:`True` any line with case control statement ends the group of lines that should be aligned and starts new group.
+#. :code:`case_control_statements_ends_group` - if set to :code:`True` any line with case control statement ends the group of lines that should be aligned and starts new group.
```

Comment:
How about his for an update:
... if set to :code:`True` any line with a case control statement (`case`, `when`, `end case`) ends the group of lines that ...

Reply:
Updated